cmd/cosign/cli/importkeypair/import_key_pair.go

```diff
@@ -36,7 +36,12 @@ func ImportKeyPairCmd(ctx context.Context, keyVal string, args []string) error {
 		return err
 	}
 
-	if cosign.FileExists("import-cosign.key") {
+	fileExists, err := cosign.FileExists("import-cosign.key\000x")
+	if err != nil {
+		return err
```

Comment:
here as well: how about we wrap the error to give more information?

thanks for working on this